python/sglang/srt/two_batch_overlap.py

```diff
@@ -341,15 +341,18 @@ def _compute_local_forward_mode(local_batch):
 
     @staticmethod
     def _compute_global_forward_mode(forward_modes):
-        converted_forward_modes = [
-            ForwardMode.DECODE.value if x == ForwardMode.IDLE.value else x
-            for x in forward_modes
+        no_idle_forward_modes = [
+            x for x in forward_modes if x != ForwardMode.IDLE.value
         ]
+
+        if not no_idle_forward_modes:
+            return ForwardMode.IDLE, True
```

Comment:
hmm, if all ranks are idle (will forward pass happesn in that case?), I would personally suggest to disable TBO since that does not look efficient and such edge case may have some bugs.

Reply:
You are right. All ranks are idle, should be disable TBO, I will refine this code.